lib/bitreserve/entities/base_entity.rb

```diff
@@ -3,15 +3,25 @@ module Entities
     class BaseEntity
       include Virtus.model
 
-      def self.from_collection(entities)
-        entities.map do |entity|
-          new(entity)
-        end
+      def self.from_collection(entities, content_range)
```

Comment:
I'm assuming it is not for now (mostly to pass the tests without having to add pagination everywhere), but maybe it can be, since they are adding pagination to all of the collection endpoints.